src/Components/Nav/Nav.jsx

```diff
@@ -2,13 +2,52 @@ import React, { Component } from 'react';
 import './Nav.scss';
 
 class Nav extends Component {
+  listener = null;
+  constructor(props) {
+    super(props);
+    this.state = {
+      colorValue: ['announceBar red', 'announceBar green', 'announceBar blue'],
+      nav: false,
+      indexNum: 0,
+    };
+  }
+  componentDidMount() {
+    window.addEventListener('scroll', this.handleScroll);
+    setInterval(() => {
+      if (this.state.indexNum >= this.state.colorValue.length - 1) {
+        return this.setState({ indexNum: 0 });
+      }
```

Comment:
return을 사용한 이유가 함수 실행을 끝내기 위해서라면 이렇게가 더 직관적일 것 같아요!
```suggestion
      if (this.state.indexNum >= this.state.colorValue.length - 1) {
        this.setState({ indexNum: 0 });
        return;
      }
```

Reply:
수정했습니다 관희님~~!!!😍